presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -164,10 +164,9 @@ private String autoFormatMetricValue(RuntimeUnit unit, long value)
 
     public void printFinalInfo()
     {
-        Duration wallTime = nanosSince(start);
-
         QueryStatusInfo results = client.finalStatusInfo();
         StatementStats stats = results.getStats();
+        Duration wallTime = Duration.succinctDuration(stats.getElapsedTimeMillis(), MILLISECONDS);
```

Comment:
This makes sense, could you please add a comment to note that this `walltime` indicates the same as the QueryStats gets?

Reply:
I can rename the variable to `serverSideWallTime` to disambiguate